d3/d3.d.ts

```diff
@@ -57,7 +57,9 @@ declare module D3 {
         x: number;
         y: number;
         keyCode: number;
-        altKey: any;
+        altKey?: boolean;
+        ctrlKey?: boolean;
+        metaKey?: boolean;
```

Comment:
I think they are for a couple of reasons:

1. https://github.com/mbostock/d3/wiki/Selections#d3_event says that "The d3.event object is a [link DOM event](https://developer.mozilla.org/en-US/docs/Web/API/Event) and implements the standard event fields". To my understanding that means that the current event can be any of the listed events, but not necessary any particular one of them.
2. They are also optional even when a keyboard event happens. For example, when a keypress of a happens, d3.event.altKey is undefined.

Ideally, I think the event should be a type union of all the possible events (since that's essentially what it is).